tabula/wrapper.py

```diff
@@ -11,12 +11,14 @@
 import io
 import json
 import os
+import platform
 import shlex
 import subprocess
-import requests
-import re
-import pandas as pd
+
 import numpy as np
+import pandas as pd
```

Comment:
Is there any reason to reorder import?

Reply:
I just used 'optimize imports' in PyCharm. It reorders imports for PEP8. You can read more about it [here](https://www.python.org/dev/peps/pep-0008/#imports)

Reply:
Also [here](https://stackoverflow.com/a/20763446) is a good answer how to works with import

Reply:
Thanks for pointing it out!